content/doc/developer/plugin-development/dependency-management.adoc

```diff
@@ -9,8 +9,9 @@ references:
 ---
 
 Plugins depend on specific versions of libraries and of other plugins, each of which is versioned independently.
-Each dependency may have dependencies of its own and different versions of different libraries and plugins may not be compatible with each other.
+Each dependency may have dependencies of its own, and different versions of different libraries and plugins may not be compatible with each other.
 Moreover, the dependencies can change with each Jenkins release, so managing these dependencies manually can be difficult and time consuming.
+Avoid using version ranges when developing plugins, as this reduces the repeatability of builds and can create problems with proper dependency management.
```

Comment:
I'd remove the word "proper" to avoid debates about various forms of dependency management.
```suggestion
Avoid using version ranges when developing plugins, as this reduces the repeatability of builds and can create dependency management problems.
```